packages/patrol/example/run_android_testlab

```diff
@@ -16,5 +14,7 @@ exec gcloud firebase test android run \
 	--results-bucket="patrol_runs" \
 	--results-dir="$RESULTS_DIR_NAME" \
 	--use-orchestrator \
-	--environment-variables clearPackageData=true,packageName="pl.leancode.patrol.example"  
-	# --device model=oriole,version=31,locale=en,orientation=portrait \
+	--environment-variables clearPackageData=true,packageName="pl.leancode.patrol.example" || TESTS_EXIT_CODE=$?
+
+echo "TESTS_EXIT_CODE=$TESTS_EXIT_CODE" >> "$GITHUB_OUTPUT"
```

Comment:
I don't like this line, because this script shouldn't know anything about GitHub Actions or any other platform.

It should be done in GitHub Actions workflow, not here.